projects/packages/stats-admin/src/class-dashboard.php

```diff
@@ -310,10 +310,6 @@ protected function get_plan_features() {
 	 * @return array An array of capabilities.
 	 */
 	protected function get_current_user_capabilities() {
-		// Feature lock.
-		if ( ! isset( $_GET['flags'] ) || $_GET['flags'] !== 'stats/ads-page' ) {// phpcs:ignore WordPress.Security.NonceVerification.Recommended
-			return array();
-		}
```

Comment:
Can you help me understand how we use this feature flag check to restrict ad dashboard access?

Reply:
That makes sense! The part I don't quite get is how we're targeting just the ads page with `get_current_user_capabilities` -- shouldn't this code block prevent all Stats pages from rendering without the `stats/ads-page` flag?

Reply:
Oh I got you. Other pages simple don't check capabilities from the frontend - code in the same link above. 😄

Reply:
Aha! Thanks for the explanation ☺️